### PR TITLE
[Bug fix] feishu-document-creator extension

### PR DESCRIPTION
**Pull request by rokcso**
## Description

A bug has been fixed. Before the fix, running the command might have launched a text editor application like Sublime Text along with opening the URL, but now, running the command will accurately select the default application for opening URLs as set by the user's system.

## Checklist

- [ ] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ ] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ ] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ ] I checked that files in the `assets` folder are used by the extension itself
- [ ] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/feishu-document-creator/src/helper.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ export default async function openFeishuURL(path: string) {
     lark: "https://www.larksuite.com",
   }[preference.appType];
   const feishuURL = host + path;
-  const defaultApplication = await getDefaultApplication(__filename);
+  const defaultApplication = await getDefaultApplication(feishuURL);
+  // console.log(`The defaultApplication is: ${defaultApplication.name}`)
```
[the remark this answers is not in the document]
Comment (rokcso) [reply extensions/feishu-document-creator/src/helper.ts]: Okay, I'm done.

File: extensions/feishu-document-creator/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Feishu Document Creator Changelog
 
+## [Bug fix] - 2024-11-21
```
Comment (LitoMore) [inline extensions/feishu-document-creator/CHANGELOG.md:3]:
```suggestion
## [Bug fix] - {PR_MERGE_DATE}
```

The Raycast bot will replace the date string once merged.

**Review by LitoMore — APPROVED** [review]

**Comment by rokcso** [comment]
@LitoMore My PR seems to have no other issues? Can we start merging the branches now? Thank you.

**Comment by LitoMore** [comment]
Hi @rokcso, you are about seven hours behind the Raycast team. Once they get back to work, the team will review your PR.

**Pull request merged**